modules/playerdata.lua

```diff
@@ -4,6 +4,7 @@ QBX = {} -- luacheck: ignore
 QBX.PlayerData = exports.qbx_core:GetPlayerData() or {}
 
 RegisterNetEvent('QBCore:Client:OnPlayerUnload', function()
+    ---@diagnostic disable-next-line: missing-fields
     QBX.PlayerData = {}
```

Comment:
Maybe we should change the definition of QBX.PlayerData to include an empty table as an allowed value for the type?

Reply:
Where is it even corely defined?